webpack.config.js

```diff
@@ -508,6 +508,7 @@ module.exports = {
                         options: {
                             sourceMap: !production,
                             sassOptions: {
+                                quietDeps: true,
```

Comment:
huh!  When I tried that it wasn't working.  I think this is new in the new sass-loader since the Webpack 5 upgrade.

Thanks in any case: this is much much nicer than pinning the version back.